test/src/scene/entityIdTest.cpp

```diff
@@ -94,13 +94,19 @@ TEST_F(EntityIdTest, BaseTest)
 		}
 	}
 
-	// Infinite loop to publish point cloud to ROS2 topic for visualization purposes. Uncomment to use. Use wisely.
-	//#ifdef RGL_BUILD_ROS2_EXTENSION
-	//    rgl_node_t ros2publishNode = nullptr;
-	//
-	//    EXPECT_RGL_SUCCESS(rgl_node_points_ros2_publish(&ros2publishNode, "pointcloud", "rgl"));
-	//    EXPECT_RGL_SUCCESS(rgl_graph_node_add_child(formatNode, ros2publishNode));
-	//    while(true)
-	//        EXPECT_RGL_SUCCESS(rgl_graph_run(raytraceNode));
-	//#endif
+// Infinite loop to publish point cloud to ROS2 topic for visualization purposes. Uncomment to use. Use wisely.
+#ifdef RGL_BUILD_ROS2_EXTENSION
+//	rgl_node_t formatNode = nullptr;
```

Comment:
Why did you put commented out macro here? Is this maybe by mistake?

Reply:
No significant reason, this style looked better to me, as the #macro creates a kind of a "box" for the commented-out code, which looks more pleasing to me :shrug:. And it's less to uncomment later, #minimalism